src/App.scss

```diff
@@ -1 +1,24 @@
-// styles go here
+.App {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.appConteiner {
+  display: flex;
+  flex-direction: column;
+  text-align: center;
+}
+
+.timer {
+  color: rgb(8, 0, 252);
+  font-size: 24px;
+}
+
+.clolck {
```

Comment:
make `clock, timer, appContainer` elements of  block `App `
use` &__clock` ... `&__timer`  (BEM)
it would be easier to identify styles you imported or mede yourself

Reply:
You are right. Fixed.